spec/tree_spec.rb

```diff
@@ -3,12 +3,73 @@
 
 describe 'Tree' do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(Tree.new.is_a? Tree).to be(true)
+  end
+
+  it '.initialize' do
+  	tree = Tree.new
+	expect(tree.height == 0).to be true
+	expect(tree.age == 0).to be true
+	expect(tree.apples == 0).to be true
+	expect(tree.alive).to be true
+  end
+  	
+  # as the tree ages, more apples grow.
+  # 2^age apples should be on the tree after it ages
+  it '.age!' do
+  	tree = Tree.new
```

Comment:
take a look at let/let!
https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let